cmd/pilot-agent/BUILD

```diff
@@ -7,6 +7,7 @@ go_library(
     visibility = ["//visibility:private"],
     deps = [
         "//cmd:go_default_library",
+        "//platform/consul:go_default_library",
```

Comment:
Why does the agent need a dep to consul ? Shouldn't the discovery handle the integration instead ?
We just managed to remove the dep of agent to k8s, please let's not add another one.